Lib/pydoc.py

```diff
@@ -183,6 +184,14 @@ def _getdoc(object):
         return None
     return inspect.cleandoc(doc)
 
+def _getversion(mod, name):
+    try:
+        return importlib.metadata.version(name)
```

Comment:
This is only safe if you know the `mod` object is the module that is going to be found by the finders, unfortunately.